### PR TITLE
Fetch exercises

### PR DESCRIPTION
**Pull request by CarlyLanglois**
fetch exercises:
1) step by step walk through of creating an html file, 
2) fetching json inside the HTML, 
3) manipulating the DOM with said json,
4) presentation of javascript's asynchronous character
5) bonus, not really fetch-critical, change display with click handler

### CODE REVIEW

File: src/chapters/fetch-json/exercises.rst [hunk 1]
```diff
@@ -85,4 +85,181 @@ JSON
...
+
+   .. sourcecode :: js
+      :linenos:
+
+      window.addEventListener("load", function() {
+         fetch("https://handlers.education.launchcode.org/static/planets.json").then(function(response){
+            response.json().then(function(json) {
+               console.log(json);
+            });
+         });
```
Comment (gildedgardenia) [inline src/chapters/fetch-json/exercises.rst]: Maybe just include lines 148-150 in the ``sourcecode`` directive. That way students who read the whole page ahead of time won't get a leg up.

File: src/chapters/fetch-json/exercises.rst [hunk 2]
```diff
@@ -85,4 +85,181 @@ JSON
...
+                  </div>
+               `;
+               index = (index + 1) % json.length;
+            });
+         });
+		});
+
+   Et voila! Our destination changes on each click!
+
+   .. figure:: ./figures/planet-destinations.gif
```
Comment (gildedgardenia) [inline src/chapters/fetch-json/exercises.rst]: The gif is broken.

Comment (CarlyLanglois) [reply src/chapters/fetch-json/exercises.rst]: hermmm

**Comment by CarlyLanglois** [comment]
@gildedgardenia addressed your notes and hopefully correctly added the gif this time!

**Pull request merged**